File: frontend/src/types/reports.ts
export interface SalesSummaryReport {
  gross_sales: string | number;
  refunds: string | number;
  net_sales: string | number;
  cogs: string | number;
  gross_profit: string | number;
}

export interface InventoryValuationReport {
  total_cost_basis: string | number;
  total_weight: string | number;
}

File: backend/app/schemas/reports.py
from pydantic import BaseModel
from decimal import Decimal

class SalesSummaryReport(BaseModel):
    gross_sales: Decimal
    refunds: Decimal
    net_sales: Decimal
    cogs: Decimal
    gross_profit: Decimal

class InventoryValuationReport(BaseModel):
    total_cost_basis: Decimal
    total_weight: Decimal
